tools/build/rules/android.bzl

```diff
@@ -105,3 +105,35 @@ android_native_app_glue = repository_rule(
         "ANDROID_NDK_HOME",
     ]
 )
+
+# Retrieve Vulkan validation layers from the Android NDK
+
+def _android_native_vulkan_validation_layers(ctx):
+    build = ""
+    for abi in ["armeabi-v7a", "arm64-v8a", "x86", "x86_64"]:
+
+        for layer in ["core_validation", "object_tracker", "parameter_validation", "threading", "unique_objects"]:
```

Comment:
If we update the NDK version (if we can), these have been renamed to be a single layer, might be worth updating GAPID for this.

Reply:
The unified layer ships in NDK r21, which is still in Beta as of today (https://developer.android.com/ndk/downloads)